templates/_templating_scripting.py

```diff
@@ -277,6 +277,15 @@ def create_issue(title: str, body: str) -> None:
                 continue
 
 
+def check_dir(args: argparse.Namespace) -> None:
+    # Ensures templates/ dir aligns with _templating_config.json.
+
```

Comment:
I'm keen to have a line similar to this one:

https://github.com/SciTools/.github/blob/5c95ee7eca6e50b56c360ab63b5c40eea6345911/templates/_templating_scripting.py#L70-L71

Just to make it clear why `args` is being passed.

Reply:
https://github.com/SciTools/.github/pull/61/commits/e47d9e12240dbc91dc530b84ef0c787516d86ee8